### PR TITLE
N15: Egregoria

### PR DESCRIPTION
**Pull request by Uriopass**
Part of #320

### CODE REVIEW

File: content/posts/newsletter-015/index.md [hunk 1]
```diff
@@ -60,6 +60,31 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+ - Island generation
+ - Day/night cycle
+ - Human AI via utilitary systems
+ - Specs to [legion 0.3][legion-github] port
+
+See also [the recent video][egregoria-video] showcasing very basic AI.
+
+_Discussions:
+[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/jkcllc/egregoria_devblog_6_island_generation_daynight/),
+[Discord][https://discord.gg/CAaZhUJ]_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-015/index.md:81]:
```suggestion
[Discord](https://discord.gg/CAaZhUJ)_
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, just a minor issue with one of the links.

File: content/posts/newsletter-015/index.md (hunk 1 above)
Comment (Uriopass) [reply content/posts/newsletter-015/index.md]: Oops :D

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:heart_eyes: (made a few small style/fmt tweaks)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**